src/utils.ts

```diff
@@ -1,22 +1,28 @@
 import { get } from 'lodash';
 import { PusherMessage } from '@pusher/chatkit-client';
-import { GiftedMessage } from './interfaces';
+import { GiftedMessage, MessagePart } from './interfaces';
 
-const getPayload = (message: PusherMessage) => get(
+const getPayload = (message: MessagePart) => get(
+  // check if type of payload is image, then do getImage
```

Comment:
TODO comments can be started with `// TODO`, some IDEs will parse it as such

Reply:
👍